### PR TITLE
fix table path for PostgreSQL

### PR DESCRIPTION
**Pull request by Zaszczyk**

### CODE REVIEW

File: scripts/Phalcon/Mvc/Model/Migration.php [hunk 1]
```diff
@@ -493,7 +493,12 @@ public static function migrate($fromVersion, $toVersion, $tableName)
...
     public function morphTable($tableName, $definition)
     {
-        $defaultSchema = self::$_databaseConfig->get('dbname');
+        if (self::$_databaseConfig->get('adapter') === 'Postgresql') {
+            $defaultSchema = self::$_databaseConfig->get('dbname') . '.' . self::$_databaseConfig->get('schema');
+        } else {
+            $defaultSchema = self::$_databaseConfig->get('dbname');
+        }
+
         $tableExists = self::$_connection->tableExists($tableName, $defaultSchema);
```
Comment (sergeyklay) [inline scripts/Phalcon/Mvc/Model/Migration.php]: See https://github.com/phalcon/cphalcon/blob/master/phalcon/db/dialect/postgresql.zep#L535

**Comment by Zaszczyk** [comment]
Did You see issue: https://github.com/phalcon/phalcon-devtools/issues/524? This pull request is connected with that.

**Comment by sergeyklay** [comment]
But this solution is not flexible enough (I mean MySQL, SQLite, etc.). 
Also please see this Postgres-like case:
```php
$config = [
    'database' => [
        'adapter'  => 'Postgresql',
        'host'     => 'localhost',
        'username' => 'username',
        'password' => '********',
        'dbname'   => 'dbname',
        'port'     => 5432,
    ],
    // ...
];
```
Without schema

**Comment by sergeyklay** [comment]
Fixed in #528

**Pull request closed without merging**